src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntryFormalizer.cs

```diff
@@ -0,0 +1,53 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Globalization;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Microsoft.Health.Dicom.Core.Features.Common;
+
+namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+{
+    public class CustomTagEntryFormalizer : ICustomTagEntryFormalizer
```

Comment:
>public [](start = 4, length = 6)

Does this need to be `public`? This goes for all classes in the PR

Reply:
**internal** also works here.  what's the benefit with that?

Reply:
Moved it to internal static method as we discussed.